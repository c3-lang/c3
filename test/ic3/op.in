quote 1 + 2
quote 1 + 2 / 3
quote 1 + 2 / 3 * 4
quote 1 + 2 / 3 * 4 - 5
1 + 2
1 + 2 / 3
1 + 2 / 3 * 4
1 + 2 / 3 * 4 - 5
