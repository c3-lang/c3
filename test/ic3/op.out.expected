quote 1 + 2
quote 1 + 2 / 3
quote 1 + 2 / 3 * 4
quote 1 + 2 / 3 * 4 - 5
3
1
4
-1
